docs/modules/clusters/pages/ucn-non-associated.adoc

```diff
@@ -0,0 +1,112 @@
+= Non-associated Namespaces
+:description: You can configure a default namespace with resources that can be used by any data structure that does not have an associated {ucn} defined, or, for example, a partition loss listener. A default namespace is also useful when you are using an API that is not namespace-aware.
+:page-enterprise: true
+:page-beta: true
+
+{description}
+
+You can use static or dynamic configuration to define your `default` namespace.
+
+== Static Configuration
+
+You can define your `default` namespace either programmatically or in the _hazelcast_ configuration file.
+
+For further information on static configuration, see the xref:clusters:ucn-static-config.adoc[] section.
+
+=== Programmatically
+
+To configure a `default` namespace programmatically, add the following lines to your code:
+
+[source,java]
+----
+UserCodeNamespaceConfig userCodeNamespaceConfig = new UserCodeNamespaceConfig();
+        userCodeNamespaceConfig.setName("default"); 
+        userCodeNamespaceConfig.addClass(loadClass("usercodenamespaces.required_class"));
+        config.getNamespacesConfig().addNamespaceConfig(userCodeNamespaceConfig);
+----
+
+=== Configuration File
+
+To configure a `default` namespace using static configuration in the _hazelcast_ configuration file, complete the following steps:
+
+. If not already open, open the _hazelcast_ configuration file in your preferred editor and scroll to the {ucn} section
+
+. Define the name of the namespace as `default`, as follows:
++
+[tabs]
+====
+XML::
++
+[source,xml]
+----
+<user-code-namespaces enabled="true">
+    :
+    <user-code-namespace name="default">
+    </user-code-namespace>
+</user-code-namespaces>
+----
+
+YAML::
++
+[source,yaml]
+----
+user-code-namespaces:
+   enabled: true
+   :
+   default:
+----
+====  
+
+. Define the resources
++
+To define a resource, you must provide an identifier for the resource, the resource type, and the location (URL) in which the resource can be found.
```

Comment:
Identifier is optional, not required (but recommended)

Reply:
I also just noticed the ZIP resources end in `.jar` when it should be `.zip`, and the `resource-type: JAR` etc entries are missing for the YAML example.